soperator/README.md

```diff
@@ -1,663 +1,168 @@
-direnv: unloading
+Edit `terraform.tfvars` with your requirements:
 
-There is no IAM token
+```hcl
+# Use your manually created jail filesystem
+filestore_jail = {
+  existing = {
+    id = "computefilesystem-<YOUR-FILESYSTEM-ID>"
+  }
+}
 
-direnv: loading <path-to>/installations/<your-installation-name>/.envrc
-token from NEBIUS_IAM_TOKEN env is used
-token from NEBIUS_IAM_TOKEN env is used
-direnv: export +AWS_ACCESS_KEY_ID +AWS_SECRET_ACCESS_KEY +NEBIUS_IAM_TOKEN +NEBIUS_PROJECT_ID +NEBIUS_TENANT_ID +NEBIUS_VPC_SUBNET_ID +TF_VAR_iam_project_id +TF_VAR_iam_tenant_id +TF_VAR_iam_token +TF_VAR_vpc_subnet_id
+# Configure GPU cluster
+k8s_cluster_node_group_gpu = {
+  resource = {
+    platform = "gpu-h100-sxm"
+    preset   = "8gpu-128vcpu-1600gb"
+  }
+  gpu_cluster = {
+    infiniband_fabric = "fabric-3"
+  }
+}
 
-IAM token is present
+# Add your SSH public key here to connect to the Slurm cluster 
+slurm_login_ssh_root_public_keys = [
+  "ssh-rsa AAAAB3N... your-key"
+]
 ```
 
-> [!TIP]
-> If you have access token expired,
-> you can switch directories back and forth to trigger unloading/loading of `.envrc` file:
-> ```shell
-> pushd .. && popd
-> ```
-> Or just simply call:
-> ```shell
-> direnv reload
-> ```
-
-> [!NOTE]
-> Instead of using `direnv`, you can just use `source` command to load variables from .envrc.
-> ```shell
-> source .envrc
-> ```
-> However, you won't get variables unloaded when you leave your installation directory.
-> I case of access token expiration, call `source` command again:
-> ```shell
-> source .envrc
-> ```
-
-Once you loaded `.envrc` file into your environment, you'll get `.aws_secret_access_key` and 
-files created in your installation directory.
-
-> [!IMPORTANT]
-> Make sure that:
-> - `.aws_secret_access_key` file is not empty
-> - `terraform_backend_override.tf` file contains valid bucket name
-
-### Configure AWS CLI
-
-Add the key, the Nebius AI region ID and the Object Storage endpoint URL to the AWS CLI configuration:
-
-```shell
-aws configure set aws_access_key_id "${AWS_ACCESS_KEY_ID}"
-```
-```shell
-aws configure set aws_secret_access_key "${AWS_SECRET_ACCESS_KEY}"
-```
-```shell
-aws configure set region 'eu-north1'
-```
-```shell
-aws configure set endpoint_url 'https://storage.eu-north1.nebius.cloud:443'
-```
+`k8s_cluster_node_ssh_access_users` is for connecting to the K8S cluster itself.
+You probably don't need this unless you want to manage the K8S cluster manually.
 
-### Initialize Terraform
 
-To initialize a Terraform project, download all referenced providers and modules, execute:
+> **Important Configuration Notes:**
+> - For large clusters: Use larger presets for CPU-only nodes
+> - Adjust storage sizes based on your needs
+> - Contact support to increase quotas if needed
+> - Ensure SSH keys are added to the correct location
 
-```shell
+5. **Deploy Your Cluster**
+```bash
 terraform init
+terraform apply # This will take ~15-20 mins
```

Comment:
It depends. But usually, it is ~40 minutes.
It waits for Slurm cluster to be up and LoadBalancer to get an IP